### ISSUE TITLE
Create a plugin for WACUP?

### ISSUE THREAD

**Issue by Dragonsong3k** [kran27/Spotiamp#4]
Not really an issue. I wasn't sure where to ask. 

I love what you have done. Simple. 

However, have you considered making this a plugin for WACUP instead? 
You have all the backend for Spotify. It would be interesting to pair it with the full functionality of WACUP to get that full Winamp experience. 

Thanks!

**Comment by kran27** [comment]
yeah replying over a year later i know, but for anyone else thinking about this, it would be best to just make something new. spotiamp is 10+ years old and using it as the base for anything new is silly. if you do want to play it through wacup/winamp, spotiamp does start up an internal shoutcast server you can connect to, but making a full integration is out of the scope of spotiamp.
